bin/main/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Root logger option
```

Comment:
Did you intend to add this file/did you add it manually? If so, why? If not, let's get rid of it -- I'm not sure what it's doing and it doesn't seem critical to this PR.